src/ethereum/shanghai/fork.py

```diff
@@ -529,6 +550,12 @@ def apply_body(
         if account_exists_and_is_empty(state, wd.address):
             destroy_account(state, wd.address)
 
+    # If any inclusion list addresses are not satisfied, the block is invalid.
+    for addr in inclusion_list_nonces:
+        sender_account = get_account(env.state, addr)
+        if inclusion_list_nonces[addr] < sender_account.nonce:
```

Comment:
It seems to me that two txs from the same sender in an IL is invalid, is this correct?

Reply:
should be fine! b/c the IL commits to the nonce and the address, so they can both be included. we just check that the nonce of the account is higher than what was specified in the IL